src/trackers/lineTracker.ts

```diff
@@ -44,6 +44,7 @@ export class LineTracker<T> implements Disposable {
 		this._selections = toLineSelections(editor?.selections);
 
 		this.notifyLinesChanged('editor');
+		this.onActiveEditorChanged?.();
```

Comment:
I was brainstorming a few alternatives to this approach.

1. Instead of the indirection through `GitLineTracker`, we could also call `this.resume()` right here. That felt strange since nothing else in `lineTracker.ts` suspends or resumes the tracker. This feels like a `gitLineTracker.ts` responsibility based on what's written today.
2. We could call `this.fireDocumentDirtyStateChanged` in the `window.onDidChangeActiveTextEditor` handler. This feels like a reasonable alternative approach to me. The main downside is that it's a bit of a slippery slope with regard to what document-specific change events are re-fired when the active editor changes. Ex: Would we also want to call `onBlameStateChanged` when the active editor changes?

I settled on the current approach, but happy to consider others in more depth if others have thoughts.